### ISSUE TITLE
Accessibility for the blind with screen readers

### ISSUE THREAD

**Issue by discapacidad5** [yakyak/yakyak#525]
Hi, I would like to move the focus with the tab key And that you can use the arrows to navigate between the them This would be very important for people who are visually impaired thanks

**Comment by averissimo** [comment]
Can you develop on that?

- tab key should change focus on which elements?
    - [ ] message input
    - [ ] message history
    - [ ] conversations 
    - [ ] create new conversation
    - [ ] others?
- Do you think a high contrast theme should be created? I don't have any experience but at first glance
    - [ ] the timestamps seem to have low contrast
    - [ ] own messages also have a not so good contrast
    - [ ] users without avatars may also have bad contrast (white over bright background)

![image](https://cloud.githubusercontent.com/assets/211358/21454087/8410393c-c90d-11e6-87d8-1271a18d4256.png)

![image](https://cloud.githubusercontent.com/assets/211358/21454129/cd116e80-c90d-11e6-83ae-8492a5aff32f.png)

**Comment by discapacidad5** [comment]
Attached is a photo that describes the accessibility you should have and
the keys you use



Can you develop on that?

   - tab key should change focus on which elements?
      -  message history
      -  conversations
      -  create new conversation
      -  others?
   - Do you think a high contrast theme should be created? I don't have any
   experience but at first glance
      -  the timestamps seem to have low contrast
      -  own messages also have a not so good contrast
      -  users without avatars may also have bad contrast (white over
      bright background)


2016-12-23 8:46 GMT-04:00 André Veríssimo <notifications@github.com>:

**Comment by averissimo** [comment]
I think attachments don't work very well when embedded in emails, can you upload via the website?

**Comment by discapacidad5** [comment]
With Mozilla Firefox and the F123 Access plugin

  What is F123 Access?

F123 Access (pronounced F 1 2 3 Akces) is a software capable of
reformatting web pages to make them more accessible and convenient for
people who are blind and using screen readers.

How can I test the F123 Access?

The software is currently available as an extension of Mozilla Firefox and
soon we will have it as an extension of Google Chrome as well. Improve the
accessibility and usability of WhatsApp on the web. F123 Access can solve
accessibility problems automatically on a large number of web pages,

Https://f123access.com/intro_en

2016-12-23 13:24 GMT-04:00 André Veríssimo <notifications@github.com>:
